Routes/adminRoute.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const Router = express.Router();
+const { signUp, login }= require('../Controller/admin_login_signup');
```

Comment:
In JS we follow `camelCase`. The file names do not contain `_`. it should be `adminLoginSignup`. This should be reflected everywhere. e.g. in Routes, `manage_kyc_details` should become `manageKycDetails`

Reply:
Did the camelCase change in the file Names